python3/koans/about_iteration.py

```diff
@@ -64,21 +64,23 @@ def is_even(item):
 
         self.assertEqual(__, even_numbers)
 
-    def test_just_return_first_item_found(self):
-        def is_big_name(item):
-            return len(item) > 4
-
-        names = ["Jim", "Bill", "Clarence", "Doug", "Eli"]
-        name = None
+    def test_filter_returns_all_items_matching_criterion(self):
+       def is_big_name(item):
```

Comment:
Both lines of this inner function (`is_big_name`) need to be indented one more space.

Reply:
embarrassing. sorry.